packer/cleanup-vhd.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash -eux
 
-## Cleanup packer SSH key and machine ID generated for this boot
+# Cleanup packer SSH key and machine ID generated for this boot
 rm -f /root/.ssh/authorized_keys
 rm -f /home/packer/.ssh/authorized_keys
 rm -f /etc/machine-id
-touch /etc/machine-id
\ No newline at end of file
+touch /etc/machine-id
+
+# Move logs under /var/log from VHD creation
+sudo mkdir /var/log.vhd
+sudo mv /var/log /var/log.vhd
```

Comment:
This puts the VHD creation logs under `/var/log.vhd/var/log`. Is that the intention, or should we just rename it?

```
sudo mv /var/log /var/log.vhd
sudo mkdir /var/log
```

Reply:
does it? My intention was to put it under `/var/log.vhd`

Reply:
It does--I think the two commands I posted are what you want.

Reply:
fixed